OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/18.3.0/MultiNodeInstall.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/python
+#!/usr/bin/env python
+
+###########################################################################################################
+
+
+# stig.py
+#
+# Copyright (c) 2012, 2018, Oracle and/or its affiliates. All rights reserved.
+#
+# NAME
+#      buildImage.py - <Build the Image>
+#
+# DESCRIPTION
+#      <This  script will build the Image>
```

Comment:
This description is insufficient. Basic instructions on how to call the script and what each parameter controls would be great.

Reply:
Add the details